ihatemoney/web.py

```diff
@@ -776,6 +789,19 @@ def edit_bill(bill_id):
 
     form = get_billform_for(g.project, set_default=False)
 
+    # Prevent editing if the bill involves deactivated user.
+    active_member_id = [m.id for m in g.project.active_members]
+    owers_id = [int(m.id) for m in bill.owers]
+    deactivated_members = set([bill.payer_id] + owers_id).difference(
+        set(active_member_id)
+    )
+    if len(deactivated_members):
+        flash(
+            _("Deactivated users involved. This bill cannot be edited."),
+            category="warning",
+        )
```

Comment:
I think having this message shown on a tooltip next to the delete or edit button of the bill would be better, don't you think?

Reply:
Good idea. A tooltip makes more sense.

Reply:
Hi Almet. I looked into the tooltip thing and found it kind of challenging to me and would take some time to finish. Would you like to keep the flashed message for now?

I'll keep working on it and launch a new pull request once I'm done.

Reply:
I prefer to merge the pull request once everything is ready :-) Take the time you need and don't hesitate to ask for help. What's blocking you?

Reply:
Thanks bro. Just some small issues. Could be done soon.

Reply:
Tooltips have been added.